overlord/aspectstate/aspecttest/mock_aspect.go

```diff
@@ -0,0 +1,34 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+/*
+ * Copyright (C) 2023 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package aspecttest
+
+// MockWifiSetupAspect returns some mocked aspect access patterns for the
```

Comment:
On "aspect access patterns"

It seems like "access patterns" and "aspects patterns" are used interchangeably? e.g.
https://github.com/snapcore/snapd/pull/12824/files#diff-552af584ab0034353f8f34499ad35458d47a63c082bfbb5ed551c54e0b3fd29dR125

https://github.com/snapcore/snapd/pull/12824/files#diff-552af584ab0034353f8f34499ad35458d47a63c082bfbb5ed551c54e0b3fd29dR127

https://github.com/snapcore/snapd/pull/12824/files#diff-552af584ab0034353f8f34499ad35458d47a63c082bfbb5ed551c54e0b3fd29dR129

and so on...

Propose we introduce a specific term/phrase for the concept of "name/path/access" (as first described in spec section `Aspects assertion header`) in the specification and use only this throughout?

Reply:
At least in my mind, that term is already "accessPattern" but, since there's a defined type with that name that would collide with a same-named variable, sometimes I also use "aspectPattern" to mean "this aspect's access patterns". However, we only use the type in `newAspect()` and not in `NewAspectBundle` (it was split at some point) so I can make it more consistent it in the latter. Thanks